### PR TITLE
Add extra to service offering

### PR DESCRIPTION
**Pull request by Ladas**
Depends on:
- [x] https://github.com/ManageIQ/topological_inventory-ingress_api-client-ruby/pull/17/

### CODE REVIEW

**Comment by Ladas** [comment]
cc @agrare

File: lib/topological_inventory/ingress_api/client/models/service_offering.rb [hunk 1]
```diff
@@ -20,6 +20,18 @@ class ServiceOffering
...
+
+    attr_accessor :documentation_url
+
+    attr_accessor :long_description
+
+    attr_accessor :distributor
+
+    attr_accessor :support_url
+
+    attr_accessor :icon_data
```
Comment (agrare) [inline lib/topological_inventory/ingress_api/client/models/service_offering.rb]: Is this right?  Shouldn't the icon be its own object type?

Comment (Ladas) [reply lib/topological_inventory/ingress_api/client/models/service_offering.rb]: right, I am removing it with the next PR, where the extra collection is brought, and this is replaced by service_offering_icon_id

Comment (agrare) [reply lib/topological_inventory/ingress_api/client/models/service_offering.rb]: Okay can you remove it from this PR?  The rest looks good.

Comment (Ladas) [reply lib/topological_inventory/ingress_api/client/models/service_offering.rb]: ah ok, I wanted to avoid manually editing this

**Pull request merged**